public-ui/src/app/views/survey-view/survey-view.component.html

```diff
@@ -1,224 +1,208 @@
 <div class="survey-view">
-
-  <div class="row">
-
-    <div class="col-xs-12">
-      <div class="page-header">
-        <div *ngIf="survey && survey.conceptId">
-          <h2> {{survey.name}} Survey </h2>
-          <h4 class="view-sub-title">{{survey.description}} </h4>
-        </div>
-      </div>
+  <div class="page-header">
+    <div class="title-container" *ngIf="survey && survey.conceptId">
+      <h1> {{survey.name}} </h1>
+      <span class="desc-container">
+        <p class="body-default info-text">{{survey.description}} </p>
+        <p class="info-text"> Click on each response to view branding logic and age and gender distributions</p>
+      </span>
     </div>
-
+  </div>
+  <div class="search-bar-container search-input-full">
+    <clr-icon shape="search" size="25" class="is-info search-icon"></clr-icon>
+    <input type="text" id="search-db" placeholder="Keyword Search" [formControl]="searchText" autofocus>
   </div>
 
-  <div class="row">
-
-    <div class="col-md-4 col-sm-6 col-xs-12">
-      <div class="white-card-box top-stat">
-        <div class="row ">
-          <div class="col-xs-12">
-            <h5 class="box-title">
-              Number Completed
-            </h5>
-            <div class="card-stat" *ngIf="!loading">
-              {{survey.participantCount | number }}
-            </div>
-            <div class="card-text">
-              partipants completed this survey
-            </div>
-          </div>
-        </div>
+  <div class="db-card">
+    <div class="survey-head">
+      <div class="stat-container">
+        <h2 class="secondary-display" *ngIf="!loading">
+          {{survey.participantCount | number }}
+        </h2>
+        <p class="info-text">
+          Participants completed this survey
+        </p>
       </div>
-    </div>
 
-    <div class="col-md-4 col-sm-6 col-xs-12">
-      <div class="white-card-box top-stat">
-        <div class="row ">
-          <div class="col-xs-12">
-            <h5 class="box-title">
-              Questions
-            </h5>
-            <div class="card-stat">
-              <span *ngIf="loading" class="spinner spinner-md"> </span>
-              <span *ngIf="!loading" >{{ surveyResult.items.length }} </span>
-            </div>
-            <div class="card-text">
-              <a href="{{surveyPdfUrl}}" download> Download Survey PDF <clr-icon shape="file" class="has-badge is-solid"></clr-icon></a>
-            </div>
-          </div>
-        </div>
+      <div class="stat-container">
+        <h2 *ngIf="!loading" class="secondary-display">
+          <span>{{ surveyResult.items.length }} </span>
+        </h2>
+        <span *ngIf="loading" class="spinner spinner-md"> </span>
+        <p class="info-text">
+          Questions
+        </p>
+      </div>
+      <div class="pdf-link">
```

Comment:
We might want to add some conditions to hide this link and other text while loading, it seems to overlap with the loading spinner.